FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/PipelineRegistrationExtensions.cs

```diff
@@ -38,6 +42,28 @@ public static class PipelineRegistrationExtensions
 
             services.AddSingleton<IGroupMemberExtractor, GroupMemberExtractor>();
 
+            FilterLocation filterLocation;
+            try
+            {
+                filterLocation = services
```

Comment:
The configuration have been validated when be registered?

Reply:
Thanks, remove double check here.